### ISSUE TITLE
running example docker locally finishes with error

### ISSUE THREAD

**Issue by maks** [GoogleCloudPlatform/functions-framework-dart#16]
I first cloned this repo and then following instructions in Readme did: `docker build -t app .` which worked thru to step 15 but then errored out:
```
Removing intermediate container 7aba22d733dd
 ---> 4cc0441c32a2
Step 12/20 : COPY . .
 ---> a146a4bcafc4
Step 13/20 : RUN mono_repo pub get --offline
 ---> Running in 63eda24cfdd8
Running `pub get --offline` across 3 package(s).

Starting `/usr/lib/dart/bin/pub get --offline` in `functions_framework`...
Resolving dependencies...
Got dependencies!
`functions_framework`: success!

Starting `/usr/lib/dart/bin/pub get --offline` in `functions_framework_builder`...
Resolving dependencies...
Warning: You are using these overridden dependencies:
! functions_framework 0.1.0 from path ../functions_framework
Got dependencies!
`functions_framework_builder`: success!

Starting `/usr/lib/dart/bin/pub get --offline` in `test/hello`...
Resolving dependencies...
Warning: You are using these overridden dependencies:
! functions_framework 0.1.0 from path ../../functions_framework
! functions_framework_builder 0.1.0 from path ../../functions_framework_builder
Got dependencies!
`test/hello`: success!
Removing intermediate container 63eda24cfdd8
 ---> 6788c1f4ee66
Step 14/20 : WORKDIR /app/test/hello
 ---> Running in 540820f58151
Removing intermediate container 540820f58151
 ---> 55fe9b2fedf1
Step 15/20 : RUN dart pub run build_runner build
 ---> Running in 89576204ccf5
[INFO] Generating build script...
[INFO] Generating build script completed, took 393ms

[INFO] Creating build script snapshot......
[INFO] Creating build script snapshot... completed, took 12.9s

[INFO] Initializing inputs
[INFO] Building new asset graph...
[INFO] Building new asset graph completed, took 936ms

[INFO] Checking for unexpected pre-existing outputs....
[INFO] Found 1 declared outputs which already exist on disk. This is likely because the`.dart_tool/build` folder was deleted, or you are submitting generated files to your source repository.

Delete these files?
1 - Delete
2 - Cancel build
3 - List conflicts
Invalid argument(s): The source must not be null
The command '/bin/sh -c dart pub run build_runner build' returned a non-zero code: 64
```

This is on Debian 10 (running in a Crostini container on ChromeOS) and 
```docker --version
Docker version 19.03.13, build 4484c46d9d
```

**Comment by subfuzion** [comment]
This happens when things get out of sync with generated outputs, especially when building locally and then inside of Docker. Adding the flag `--delete-conflicting-outputs` normally fixes this by telling the build_runner to assume conflicting outputs are from previous builds, and to just skip the prompt and delete the outputs. This should be fixed with #23.

**Comment by maks** [comment]
Brillant, just pulled latest master and works perfectly, thanks for the quick fix @subfuzion !
